### DISCUSSION TITLE
shortcut to expand/collapse all nodes in editor scenetree

### DISCUSSION THREAD

**Discussion by AndHalfAFish** [godotengine/godot-proposals#4527] [Editor]
In Blender for example, using shift+A in the outliner toggles between showing the entire node hierarchy & collapsing it. 
I would love something similar for the nodes in the editor scenetree in Godot. It makes it easier to find objects without having to search and type.

Illustration from Blender (disregard the material & mesh nodes)

![toggle_node_hierarchy](https://user-images.githubusercontent.com/59080934/167799663-4f004f0e-fcd2-45b9-b6bd-8dbf7978b46b.png)

**Comment by recore67** [comment]


**Reply by AThousandShips** [reply]
You can already do this by holding down shift while clicking

**Reply by recore67** [reply]
oh wow i had no clue, nevermind then lol

**Comment by AThousandShips** [comment]
There already is such an option in the scene tree menu, and you can add a shortcut to it under `scene_tree/expand_collapse_all` in the editor settings (this is already in 3.x)
